contrib/init/bitcoind.service

```diff
@@ -19,7 +19,26 @@ User=bitcoin
 Type=forking
 PIDFile=/run/bitcoind/bitcoind.pid
 Restart=on-failure
+
+# Hardening measurements
```

Comment:
Nit: Hardening measures, not measurements :)

Reply:
Thanks for pointing this out. Fixed with 8748949.